app/src/docs/v1.api-spec.yaml

```diff
@@ -1391,6 +1443,90 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /rbac/submissions:
+    get:
+      summary: Get a list of users and permissions for a submission
+      operationId: getSubmissionUsers
+      tags:
+        - RBAC
+      parameters:
+        - in: query
+          name: formSubmissionId
+          schema:
+            type: string
+            format: uuid
+            example: c6455376-382c-439d-a811-0381a012d696
+          description: submission ID
+          required: true
+        - in: query
+          name: userId
+          schema:
+            type: string
+          description: user ID
+      responses:
+        "200":
+          description: "OK"
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/UserAccessSubmission"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+    put:
+      summary: Set premissions for a user on a submission
+      operationId: setSubmissionUser
+      tags:
+        - RBAC
+      parameters:
+        - in: query
+          name: formSubmissionId
+          schema:
+            type: string
+            format: uuid
+            example: c6455376-382c-439d-a811-0381a012d696
+          description: submission id
+          required: true
+        - in: query
+          name: userId
+          schema:
+            type: string
+          description: user ID
+          required: true
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                type: string
+                example: "SUBMISSION_UPDATE"
+              description: an array of the sumbission permissions to apply to this user. [] to remove.
```

Comment:
May be better to say "set an empty array to remove" instead?

Reply:
👍